### ISSUE TITLE
Azure Striker, Cultcept, Dementium & Others

### ISSUE THREAD

**Issue by Adlerschwinge** [Smumbo/DefinitiveIcons#9]
Azure Striker Gunvolt Badges (maybe GVs or Copens Emblems? Or organizations like quill/Eden?),  

Other (smaller) suggestions; 
Cultcept (Dice/(Anniversary) Logo)
dementium (steam emotes seem pretty nice :D)
Devils and realist (sigils from dantalion/sytry?)
Maybe something related to style savvy/new style boutique? Or fashion/dress up games in general 

Sorry if it's a bit much, i feel like those would be really nice additions!

**Comment by Smumbo** [comment]
Hey, thanks for the suggestions!

Since Devils and Realist only consists of one game, I don't want to include that one, but I've added all of your other suggestions in [v3.2](https://github.com/Smumbo/DefinitiveIcons/releases/tag/v3.2).

I'm not super happy with the Dementium one; all of my ideas I tried look compromised with the small size of these icons, so I just traced the Steam emote. If you have any other ideas for Dementium, I might want to replace it.

Also, I really liked your idea for fashion games, so I added icons for Style Savvy and the fashion game genre in general.
